### ISSUE TITLE
Stuck at the "Get token!" screen. Submit Button not functioning

### ISSUE THREAD

**Issue by JTDaddy17** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#88]
I click on submit when I paste the token, but nothing happens. It doesn't show me the permissions screen. Just, nothing happens... please fix this: 
![image](https://user-images.githubusercontent.com/50080776/107711943-86f48c80-6c96-11eb-94a2-9935b0467458.png)

**Comment by ViniciusR** [comment]
Same here!

**Comment by Melvin-Abraham** [comment]
@JTDaddy17 @ViniciusR An error screen prior to the "Get Token" screen might have popped up:

![image](https://user-images.githubusercontent.com/26056183/107726699-ada2d980-6d0f-11eb-8739-898f6e4e7add.png)

After the above screen is ignored, the submit button does not work (_Event Listeners_). Although this error is fixed in the source, it's not released yet. So, the following should help at the moment:

> Fixed in dd67747. Since, this fix will not be readily available until next release ... you can _temporarily_ disable **Settings** > **Conversation** > **Enable microphone on application startup**

**Ref:** https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/39#issuecomment-734111970

**Comment by JTDaddy17** [comment]
![image](https://user-images.githubusercontent.com/50080776/107731054-f7b19e00-6cc2-11eb-8e6a-2aa54598d396.png)
I did that but a new issue has shown up

**Comment by Melvin-Abraham** [comment]
@JTDaddy17 That's a rate limit issue related to Google Cloud. You can try that again the next day and it should work...

**Comment by TheNexter** [comment]
> ![image](https://user-images.githubusercontent.com/50080776/107731054-f7b19e00-6cc2-11eb-8e6a-2aa54598d396.png)
> I did that but a new issue has shown up

Delete folder : inside %appdata%
uninstall the app

Reinstall them
And now it's working

**Comment by JTDaddy17** [comment]
> > ![image](https://user-images.githubusercontent.com/50080776/107731054-f7b19e00-6cc2-11eb-8e6a-2aa54598d396.png)
> > I did that but a new issue has shown up
> 
> Delete folder : inside %appdata%
> uninstall the app
> 
> Reinstall them
> And now it's working

Did that too, still no dice

**Comment by JTDaddy17** [comment]
> @JTDaddy17 That's a rate limit issue related to Google Cloud. You can try that again the next day and it should work...

Still bringing up the same error

**Comment by rbrouwer** [comment]
I believe these might be an issue if writing to the "Saved Tokens Path" requires admin rights. In that case it might actually help to run it as admin once to get past this!

**Comment by anthonybaldwin** [comment]
> I believe this might be an issue if writing to the "Saved Tokens Path" requires admin rights. In that case it might actually help to run it as admin once to get past this!

Yep. That's it...and why only _some_ people had luck re-installing.

**Solution:** Run as admin, or don't place the config in a location that requires administrative access to use this.

**Comment by StevenJonSmith** [comment]
Moved my auth and token files from the C drive program install location to my %appdata% folder and all is now working well without having to run as admin. Thank you!

**Comment by Adamis9876** [comment]
> I click on submit when I paste the token, but nothing happens. It doesn't show me the permissions screen. Just, nothing happens... please fix this:
_ am having the same issue and the error message at startup is "Tried calling start() before the ready event!"

**Comment by morgenman** [comment]
Admin didn't fix for me, nor moving the location, nor did the mic thing. I had to delete the %appdata% folder and set it up again.
